EDScoutWebUI/templates/index.html

```diff
@@ -218,35 +218,38 @@
                 container.prepend(newBodyEntry);
             }
 
-            function createIconElement(iconType, elementType='i') {
+            function createIconElement(iconType, description='', elementType='i') {
                 let iconElement = document.createElement(elementType)
                 iconElement.classList.add("flaticon-"+iconType)
+                if (description != "") {

```

Comment:
I'm no java script expert but I think it might be safer here to test using `!==`?

https://stackoverflow.com/questions/8616483/javascript-comparison-operators-vs

Also, the default uses `''` whereas we test against `""`. Functionally the same I think so could probably be left as-is?

Reply:
I'm no javascript expert either (is anybody, really?), but these sound like good things to do. Update coming..